src/main/java/org/springframework/samples/petclinic/model/Visit.java

```diff
@@ -61,6 +61,19 @@
     @Column(name = "description")
     private String description;
 
+    @Column(name = "scheduled")
+    private Boolean scheduled;
```

Comment:
Since we do not allow null values, preferably we can use the primitive boolean. 
Also we can add a default value